protocol/messenger_communities.go

```diff
@@ -458,6 +458,26 @@ func (m *Messenger) RequestToJoinCommunity(request *requests.RequestToJoinCommun
 		}
 	}()
 
+	// Activity Center notification
+	message := &common.Message{}
+	message.CommunityID = request.CommunityID.String()
+	message.Text = fmt.Sprintf("Request to join community %s", community.Name())
+
+	notification := &ActivityCenterNotification{
+		ID:        types.FromHex(message.ID),
```

Comment:
ID will be empty, you need to set the ID of the community to the ID of the request I guess